src/main/java/fr/zcraft/quartzlib/components/i18n/translators/gettext/POFile.java

```diff
@@ -64,6 +64,7 @@
 
     private Integer pluralCount = 2;
     private String pluralFormScript = "";
+    private PluralForms pluralForms = null;
```

Comment:
Setting the pluralForms as final maybe ?

Reply:
Impossible: the field is only set when the file is parsed (as we don't know what to set before). `final` fields without default value must be set in the constructor.